src/types.ts

```diff
@@ -30,6 +30,8 @@ export interface ThemeProviderProps {
   enableColorScheme?: boolean
   /** Key used to store theme setting in localStorage */
   storageKey?: string
+  /** Name used to store active theme as a cookie (for use in React Server Components). If undefined, no cookies will be set */
+  cookieName?: string
```

Comment:
does this do anything if there's no ServerThemeProvider? If not, we should mention that. Also, it kind of reads like "if you're in a client component you don't need to provide this", but you do need to provide the same value as in the ServerThemeProvider, right?